### ISSUE TITLE
feat(CLI): Add a demo command

### ISSUE THREAD

**Issue by gaocegege** [tensorchord/envd#706]
## Description

`envd demo` creates a build.envd demo and up it.

---

**Message from the maintainers**:

Love this enhancement proposal? Give it a 👍. We prioritise the proposals with the most 👍.

**Comment by kemingy** [comment]
Can we support `envd up -f <URL>` to build and run the remote envd file?

**Comment by gaocegege** [comment]
SGTM but the build context is still a problem. We cannot get the remote build context from the URL.

**Comment by VoVAllen** [comment]
Probably we can make this part of `envd init` command, such as `envd init --demo`

**Comment by gaocegege** [comment]
In my opinion, the command `envd demo` should create the build.envd in tmp/xx/ then run `envd up` based on this. Then the users can get a shell from it directly.

**Comment by aseaday** [comment]
How about we provide a interfact shell like:
```bash
$ envd demo
> Which one do you want to use: Python/R/Julia
The following packages will be installed for you:
- numpy
- 
> What packages do you want install extra
```

**Comment by gaocegege** [comment]
Looks promising to me. :tada:

**Comment by gaocegege** [comment]
It also can be used in `envd init`

**Comment by aseaday** [comment]
> It also can be used in `envd init`

yes, we could had a feature to support interactive set simple envd build manifest first.

**Comment by kemingy** [comment]
interactive shell:

* https://github.com/charmbracelet/gum

**Comment by AlexXi19** [comment]
This would be helpful, an interactive shell to choose a simple initial configuration.

**Comment by gaocegege** [comment]
I marked the issue priority/medium

**Comment by AlexXi19** [comment]
Interested in this issue, could use this library: https://github.com/charmbracelet/bubbletea

**Comment by gaocegege** [comment]
The lib LGTM. Please pick it up if you are interested in it!
